### ISSUE TITLE
Build warning

### ISSUE THREAD

**Issue by vyky** [aksonov/react-native-tableview#128]
Build warning with react-native 0.39, xcode 8.2, ios 10.2

```
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:70:1: warning: method possibly missing a [super insertReactSubview:atIndex:] call [-Wobjc-missing-super-calls]
}
^
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:291:23: warning: 'sendInputEventWithName:body:' is deprecated: Use RCTDirectEventBlock or RCTBubblingEventBlock instead [-Wdeprecated-declarations]
    [_eventDispatcher sendInputEventWithName:@"onWillDisplayCell" body:@{@"target":self.reactTag, @"row":@(indexPath.row), @"section": @(indexPath.section)}];
                      ^
In file included from /Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:11:
../react-native/React/Base/RCTEventDispatcher.h:90:1: note: 'sendInputEventWithName:body:' has been explicitly marked deprecated here
- (void)sendInputEventWithName:(NSString *)name body:(NSDictionary *)body
^
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:295:23: warning: 'sendInputEventWithName:body:' is deprecated: Use RCTDirectEventBlock or RCTBubblingEventBlock instead [-Wdeprecated-declarations]
    [_eventDispatcher sendInputEventWithName:@"onEndDisplayingCell" body:@{@"target":self.reactTag, @"row":@(indexPath.row), @"section": @(indexPath.section)}];
                      ^
In file included from /Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:11:
../react-native/React/Base/RCTEventDispatcher.h:90:1: note: 'sendInputEventWithName:body:' has been explicitly marked deprecated here
- (void)sendInputEventWithName:(NSString *)name body:(NSDictionary *)body
^
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:391:48: warning: format specifies type 'id' but the argument has type 'int _Nullable' [-Wformat]
            NSLog(@"selectedAccessoryType %@", [item[@"selectedAccessoryType"] intValue]);
                                          ~~   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                          %d
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:464:23: warning: 'sendInputEventWithName:body:' is deprecated: Use RCTDirectEventBlock or RCTBubblingEventBlock instead [-Wdeprecated-declarations]
    [_eventDispatcher sendInputEventWithName:@"press" body:newValue];
                      ^
In file included from /Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:11:
../react-native/React/Base/RCTEventDispatcher.h:90:1: note: 'sendInputEventWithName:body:' has been explicitly marked deprecated here
- (void)sendInputEventWithName:(NSString *)name body:(NSDictionary *)body
^
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:474:23: warning: 'sendInputEventWithName:body:' is deprecated: Use RCTDirectEventBlock or RCTBubblingEventBlock instead [-Wdeprecated-declarations]
    [_eventDispatcher sendInputEventWithName:@"onAccessoryPress" body:newValue];
                      ^
In file included from /Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:11:
../react-native/React/Base/RCTEventDispatcher.h:90:1: note: 'sendInputEventWithName:body:' has been explicitly marked deprecated here
- (void)sendInputEventWithName:(NSString *)name body:(NSDictionary *)body
^
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:488:23: warning: 'sendInputEventWithName:body:' is deprecated: Use RCTDirectEventBlock or RCTBubblingEventBlock instead [-Wdeprecated-declarations]
    [_eventDispatcher sendInputEventWithName:@"change" body:@{@"target":self.reactTag, @"sourceIndex":@(sourceIndexPath.row), @"sourceSection": @(sourceIndexPath.section), @"destinationIndex":@(destinationIndexPath.row), @"destinationSection":@(destinationIndexPath.section), @"mode": @"move"}];
                      ^
In file included from /Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:11:
../react-native/React/Base/RCTEventDispatcher.h:90:1: note: 'sendInputEventWithName:body:' has been explicitly marked deprecated here
- (void)sendInputEventWithName:(NSString *)name body:(NSDictionary *)body
^
/Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:507:27: warning: 'sendInputEventWithName:body:' is deprecated: Use RCTDirectEventBlock or RCTBubblingEventBlock instead [-Wdeprecated-declarations]
        [_eventDispatcher sendInputEventWithName:@"change" body:newValue];
                          ^
In file included from /Users/user/app/node_modules/react-native-tableview/RNTableView/RNTableView.m:11:
../react-native/React/Base/RCTEventDispatcher.h:90:1: note: 'sendInputEventWithName:body:' has been explicitly marked deprecated here
- (void)sendInputEventWithName:(NSString *)name body:(NSDictionary *)body
^
8 warnings generated.
```

**Comment by iRoachie** [comment]
These warnings should be fixed in the next release. Partial commit here https://github.com/aksonov/react-native-tableview/pull/149/commits/1b5a8b31bfe48ae4fce4e4556b36ba3b57c012ce

**Comment by iRoachie** [comment]
Fixed and merged in master. Will be available in 2.0
